app/src/main/java/com/example/drawingactivity/ColorWheel.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.drawingactivity
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.util.Log
+import android.view.View
+
+@SuppressLint("ClickableViewAccessibility")
+class ColorWheel @JvmOverloads constructor(
```

Comment:
ColorWheel makes me think it's a picker color view. Is this the case? This is just the color circle right?

Reply:
Good callout, I have renamed it to PrimaryColorCircleView as it is more accurate to what it is